glide.yaml

```diff
@@ -0,0 +1,18 @@
+package: github.com/Clever/resolve-ip
+import:
+- package: github.com/davecgh/go-spew
+  version: 5215b55f46b2b919f50a1df0eaa5886afe4e3b3d
+  subpackages:
+  - spew
+- package: github.com/pmezard/go-difflib
+  version: d8ed2627bdf02c080bf22230dbb337003b7aba2d
+  subpackages:
+  - difflib
+- package: github.com/stretchr/testify
+  version: 67106a5111a06241c8d84952c33214675f51a34a
+  subpackages:
+  - assert
+- package: gopkg.in/Clever/optimus.v3
+  version: adba1ec91df15f8b00c97a62e5ea6c348b3e0b7d
+  subpackages:
+  - sources/csv
```

Comment:
can you run `glide config-wizard`? I suspect we can use semver for things like optimus